src/TestProblem/Hydro/Riemann/Init_TestProb_Hydro_Riemann.cpp

```diff
@@ -57,6 +57,18 @@ static double    Riemann_YeR;          // right-state electron fraction
 // =======================================================================================
 
 
+// problem-specific function prototypes
+#if ( EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE=NUC_TABLE_MODE_TEMP )
+bool   Flu_ResetByUser_Func_Riemann( real fluid[], const double x, const double y, const double z, const double Time,
+                                 const double dt, const int lv, double AuxArray[] );
+void   Flu_ResetByUser_API_Riemann( const int lv, const int FluSg, const double TimeNew, const double dt );
+
+// this test problem needs to reset both Flu_ResetByUser_API_Ptr and Flu_ResetByUser_Func_Ptr, while
```

Comment:
Only the `NUCLEAR1` and `NUCLEAR2` need to reset these two pointers.

Reply:
I revised the comment a bit.